### PR TITLE
Alerts in Beta

### PR DESCRIPTION
**Pull request by summerscope**
Screenshot 2024-09-26 at 14 25 38

### CODE REVIEW

**Review by ChristopherGS — APPROVED** [review]

File: docs/guides/web-ui/alerts.md [hunk 1]
```diff
@@ -42,7 +45,7 @@ The **Webhook URL** field is where you can specify a URL to which the alert will
 For now, **Logfire** alerts only send the requests in [Slack format].
 
 ??? tip "Get a Slack webhook URL"
-    To get a Slack webhook URL, follow the instructions in the [Slack documentation](https://api.slack.com/messaging/webhooks).
+To get a Slack webhook URL, follow the instructions in the [Slack documentation](https://api.slack.com/messaging/webhooks).
```
Comment (alexmojaki) [inline docs/guides/web-ui/alerts.md]: this spacing was intentional to make the tip render properly

Comment (alexmojaki) [reply docs/guides/web-ui/alerts.md]: before:

Screenshot 2024-09-26 at 14 32 57

after:

Screenshot 2024-09-26 at 14 33 00

Comment (summerscope) [reply docs/guides/web-ui/alerts.md]: Sorry I think my auto-formatter did that. Was there a reason to make this admonition an accordion @alexmojaki if you want it to be expanded by default? It makes more sense to me to have this be static (not collapsable) in this case, and use the accordion state when you want it collapsed by default.

Comment (Kludex) [reply docs/guides/web-ui/alerts.md]: I don't think it was expanded by default when it was created.... 🤔

Comment (alexmojaki) [reply docs/guides/web-ui/alerts.md]: it's not expanded by default. look at https://logfire.pydantic.dev/docs/guides/web-ui/alerts/#create-an-alert and compare with the preview.

Comment (summerscope) [reply docs/guides/web-ui/alerts.md]: sorry @alexmojaki I got myself confused looking at your screengrab (thinking that showed the default state), I'll fix this up.

**Pull request merged**